js/src/name/resolve.js

```diff
@@ -83,12 +81,12 @@ module.exports = (createCommon, options) => {
 
             done()
           })
-        }, 1)
+        }, 10)
```

Comment:
Rather than having this is a timeout, we should have something similar to this: https://github.com/ipfs/interface-ipfs-core/blob/master/js/src/pubsub/utils.js#L5-L31

Where we execute the expectations once every 100 milliseconds until either it passes or we've tried 10 times. This will make the test less likely to fail on lower performance machines.